### PR TITLE
ref(Reprocessing): Fetch group every 30s to update reprocessing progressbar

### PR DESCRIPTION
**Pull request by priscilawebdev**
**Description:**

When reprocessing an event, the user must always click on the "Refresh" button to fetch the group data (again) and obtain the new status of the activity. Depending on the new status of the activity, the user will be redirected to the page of the new reprocessed event.

image

The problem is that always clicking the button can be a little boring, so this PR improves that experience a little. Now, the pi group will be obtained every 30 seconds if the status is equal to reprocessing.

At the moment we don't make use of Websockets to listen to the server when reprocessing an event, but that is something we want to implement in the future.

### CODE REVIEW

File: src/sentry/static/sentry/app/views/organizationGroupDetails/groupDetails.tsx [hunk 1]
```diff
@@ -163,97 +172,182 @@ class GroupDetails extends React.Component<Props, State> {
...
+    try {
+      const eventPromise = this.canLoadEventEarly(this.props)
+        ? this.getEvent()
+        : undefined;
+
+      const groupPromise = await api.requestPromise(this.groupDetailsEndpoint, {
+        query: this.getGroupQuery(),
+      });
+
+      const [data] = await Promise.all([groupPromise, eventPromise]);
```
Comment (matejminar) [inline src/sentry/static/sentry/app/views/organizationGroupDetails/groupDetails.tsx:340]: `groupPromise` gets assigned to`data` variable, what about eventPromise? Is that just some sort of side-effect?

**Review by matejminar — COMMENTED** [review]
30 seconds sounds like a lot, would re-fetching more often (like 10 secs) cause performance issues?

File: src/sentry/static/sentry/app/views/organizationGroupDetails/groupDetails.tsx (hunk 1 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/views/organizationGroupDetails/groupDetails.tsx:340]: oh I did not write this code and did not want to change it in this PR

**Comment by priscilawebdev** [comment]
@matejminar yes 30s sounds like a lot, but as the reprocessing takes minimum 10s to reprocess, Markus said that 30.000 ms would be a good time.

**Review by matejminar — APPROVED** [review]

**Pull request merged**